decentraland/unity-explorer
# feat: /goto crowd command and refactor

## What does this PR change?



This PR adds the /goto crowd command with a refactor to the commands system (removing regex usage primarily). Additionally the output of /help will now include debug only commands when we are in debug mode.

## How to test the changes?



1. Launch the explorer
2. Type in /goto crowd - should teleport you to the most crowded spot in Genesis
3. Check that all other commands work
4. Ensure that debug only commands (like loadpx, killpx, debug) are only printed in /help when running the client with --debug.

Commands to test:

- /world
- /clear
- /debug
- /goto
- /goto-local
- /help
- /killpx
- /loadpx
- /reload
- /show-entity
- /world

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
